components/datatypes/agentic/conversationsignals.schema.json

```diff
@@ -0,0 +1,84 @@
+{
+  "meta:license": [
+    "Copyright 2025 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/agentic/conversationsignals",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Conversation Signals",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Conversation entity signals extracted from raw text. Raw text can be from a prompt, response, feedback, etc.",
```

Comment:
@rbuck23 it would be good to add which fields are required, the examples help, but it is not clear which fields are mandatory.